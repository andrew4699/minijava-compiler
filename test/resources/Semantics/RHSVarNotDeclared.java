class A {

  public static void main(String[] args) {
    System.out.println(new B().b());
  }
}

class B {
  boolean y;

  public boolean b() {
    y = true && x;
    return false;
  }
}
