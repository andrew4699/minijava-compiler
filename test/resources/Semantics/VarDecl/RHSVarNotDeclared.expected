[VarDeclVisitor - 12]: x undeclared at B.b
